helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -370,10 +370,8 @@ internal fun OrtResult.getPackageOrProject(id: Identifier): Package? =
 internal fun OrtResult.getProvenance(id: Identifier): Provenance? {
     val pkg = getPackageOrProject(id)!!
 
-    scanner?.results?.scanResults?.forEach { (_, results) ->
-        results.forEach { scanResult ->
-            if (scanResult.provenance.matches(pkg)) return scanResult.provenance
-        }
+    getScanResultsForId(id).forEach { scanResult ->
+        if (scanResult.provenance.matches(pkg)) return scanResult.provenance
```

Comment:
As `getScanResultsForId()` is used, the if condition will always be true and thus is redundant.